options/ansi/include/stdlib.h

```diff
@@ -115,6 +115,7 @@ size_t wcstombs(char *__restrict __mb_string, const wchar_t *__restrict __wc_str
 
 #if __MLIBC_GLIBC_OPTION
 typedef int (*comparison_fn_t) (const void *__a, const void *__b);
+int rpmatch(const char *resp);
```

Comment:
Please do not continue this broken tradition of inlining glibc stuff in the ANSI stdlib.h!

Make a `stdlib.h` for the glibc option and put it there.